src/elm/Page/Shop.elm

```diff
@@ -303,7 +301,7 @@ viewCard model ({ shared } as loggedIn) card =
         ]
         [ div [ class "md:hidden rounded-lg bg-white h-32 flex" ]
             [ div [ class "w-1/4" ]
-                [ img [ class "rounded-l-lg object-cover h-32 w-full", src imageUrl ] []
+                [ img [ class "rounded-l-lg object-cover h-32 w-full", src image, on "error" (Json.Decode.succeed (OnImageError index)) ] []
```

Comment:
Didn't know that we can handle image source loading errors this way 👍

Reply:
This was news for me as well. I was hoping there would be an HTML-only way, but apparently the only one that exists involves a bit of hacking with the `object` tag. This is the next best thing :p

Reply:
wow this is actually pretty neat